go/engine/device_keygen.go

```diff
@@ -191,6 +239,11 @@ func (e *DeviceKeygen) localSave(ctx *Context) {
 	if e.runErr = e.naclEncGen.SaveLKS(e.G(), e.args.Lks, ctx.LoginContext); e.runErr != nil {
 		return
 	}
+	if e.naclSharedDHGen != nil {
+		if e.runErr = e.naclSharedDHGen.SaveLKS(e.G(), e.args.Lks, ctx.LoginContext); e.runErr != nil {
```

Comment:
I don't want this key to his local storage, so I think we can leave this out. The plan is going to be to always sync from the server on startups.